test/tx_build/change_trust_test.exs

```diff
@@ -0,0 +1,53 @@
+defmodule Stellar.TxBuild.ChangeTrustTest do
+  use ExUnit.Case
+
+  import Stellar.Test.XDRFixtures, only: [change_trust_xdr: 2]
+
+  alias Stellar.TxBuild.{Amount, Asset, ChangeTrust}
+
+  setup do
+    asset_code = "BTCN"
+    asset_issuer = "GBTG2POJVVSRBQSZVA3IYJEZJQLPTIVVYOYRLTZEAEFBMVP72ZTQYA2V"
+    amount = 100
+
+    %{
+      asset: {asset_code, asset_issuer},
+      amount: amount,
+      xdr: change_trust_xdr({asset_code, asset_issuer}, amount)
```

Comment:
What about using a literal XDR binary instead of building one on the fly?. We could create the XDR binary with one of the official libraries and use it in our tests. My concern is that a bug will be way more challenging to detect by making the binaries on the fly.

Reply:
@luishurtado yep makes too much sense!! I'll schedule this to refactor.

Note: we match structures from the StellarBase library here.

Best!!